packages/interfaces/src/relay.ts

```diff
@@ -15,6 +15,7 @@ export interface IRelayAPI {
   readonly gossipSub: GossipSub;
   start: () => Promise<void>;
   getMeshPeers: (topic?: TopicStr) => PeerIdStr[];
+  getAllMeshPeers: () => Map<TopicStr, PeerIdStr[]>;
```

Comment:
I don't believe we need to add that, I think it's fair to assume the caller know what pubsub topic they are interested in.

https://github.com/waku-org/js-waku/pull/1586/files#r1337929607